### DISCUSSION TITLE
Unable to submit solution

### DISCUSSION THREAD

**Discussion by lucasP243** [cssbattle/forum#101] [Bugs reports]
Hi,

For some reason, I am unable to sumbit any solution. 
After a click on the submit button, it remains in its disabled state and the console prints out the following error message:
![image](https://user-images.githubusercontent.com/58945530/132931571-8020b560-88cd-4f90-bae2-7df31cbe75de.png)

I don't think there is any problem with the code I tried to submit, since I also tried to submit solutions I validated a few months ago and it did not work either.

**Comment by chinchang** [comment]
hi! are you logged in? Also, which browser?

**Comment by lucasP243** [comment]
Hi, thanks for your reply

After trying again just now, the problem seems to be fixed
Still, just FYI, yes I am logged in as @lucas_p243, and I am using Chrome 93
